components/renku_data_services/storage_schemas/api.spec.yaml

```diff
@@ -47,9 +148,112 @@ components:
       type: object
       properties:
         git_url:
+          $ref: "#/components/schemas/GitUrl"
+        storage_type:
+          type: string
+          description: same as rclone prefix
```

Comment:
This specifically refers to prefix in the rclone schema, e.g.
```
   {
[...]
        "Name": "s3",
        "Description": "Amazon S3 Compliant Storage Providers including AWS, Alibaba, ArvanCloud, Ceph, China Mobile, Cloudflare, GCS, DigitalOcean, Dreamhost, Huawei OBS, IBM COS, IDrive e2, IONOS Cloud, Leviia, Liara, Lyve Cloud, Minio, Netease, Petabox, RackCorp, Scaleway, SeaweedFS, StackPath, Storj, Synology, Tencent COS, Qiniu and Wasabi",
        "Prefix": "s3",
        "Options": [
            {
                "Name": "provider",
                "Help": "Choose your S3 provider.",
                "Provider": "",
                "Default": "",
                "Value": null,
                "Examples": [
                    {
                        "Value": "AWS",
                        "Help": "Amazon Web Services (AWS) S3",
                        "Provider": ""
                    },
[...]
  ```

Reply:
But still do we need to ask users to come up with this name rather than assign a unique id ourselves? We probably will have to enforce uniqueness on these anyhow. So why now assign the ulid to this field and keep it more internal?

Reply:
I'll move it so it uses the `configuration.type` field directly and doesn't need to be specified a second time. I'm not sure about your ULID comment, a project could have multiple e.g. "s3" storages and an ULID would just be harder to read/understand as opposed to "s3" or "azureblob"